src/application/application.js

```diff
@@ -16,12 +14,6 @@ export default Application.extend({
     this.$body = $(document.body);
     this.layout = new LayoutView();
     this.layout.render();
-
-    this.listenTo(routerChannel, {
-      'before:enter:route' : this.onBeforeEnterRoute,
-      'enter:route'        : this.onEnterRoute,
-      'error:route'        : this.onErrorRoute
-    });
```

Comment:
Humm... we need these events to handle things like `nprogress` in this case, right? Some idea about how to proceed to replace it? Maybe bubbling route events to `Application` in some way?